explorer/app/viewModelBuilders/catalog/altos-labs-catalog/common/viewModelBuilders.ts

```diff
@@ -232,3 +260,24 @@ function getCatalogBreadcrumbs(
   }
   return breadcrumbs;
 }
+
+/**
+ * Returns the S3 url for the specified URI pathname.
+ * @param processedData - Boolean indicating whether data is processed.
+ * @returns the S3 url.
+ */
+function getS3Uri(processedData: boolean): string {
+  if (processedData) {
```

Comment:
Can we check the existence of a file name instead of using the processed/raw? Processed/raw may not always hold true. The main difference is if its a path or a file.

Reply:
Hi @NoopDog the check refers to the function `isProcessedData` where the filename ends with `.h5ad`. For clarity, I will reword `processedData`.

Reply:
Hi @NoopDog, I'm sorry, I totally missed the file/path difference!